packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -1352,14 +1365,41 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | TeamsCa
 
   /* @conditional-compile-remove(breakout-rooms) */
   private assignedBreakoutRoomUpdated(breakoutRoom: BreakoutRoom): void {
+    if (this.call?.id === undefined) {
+      return;
+    }
```

Comment:
Look into why need this.

Reply:
Tested the bug bash scenarios after removing this and everything passes.

This condition might have actually caused some test cases to fail in the initial bug bash when a breakout rooms call ends (when closed or timed out) and the tester did not come back to the meeting.